bioimageio_chatbot/chatbot_extensions/hpa_extension.py

```diff
@@ -0,0 +1,112 @@
+from bioimageio_chatbot.utils import ChatbotExtension
+from schema_agents import schema_tool
+from pydantic import Field, BaseModel
+from typing import Optional, List, Dict, Any
+import pandas as pd
+from pathlib import Path
+import requests
+import re
+import os
+from bioimageio_chatbot.utils import download_file
+
+class HPAClient:
+    def __init__(self):
+        self._base_url = 'https://www.proteinatlas.org/download/proteinatlas.tsv.zip'
+        file_path = Path('./data/proteinatlas.tsv.zip')
+        # firstly check if the data is already downloaded in the /data folder
+        if not os.path.exists(file_path):
+            # download the data
+            download_file(self._base_url, file_path)
```

Comment:
Make sure ./data existis before downloading

Reply:
add os.makedirs to make sure